lib/fake_subscription_data.csv

```diff
@@ -0,0 +1,414 @@
+1,296512
```

Comment:
Is there any chance the commit for this could explain what this data is and where it came from?

Reply:
Agreed, added to 02ee34a.